demo/owner/README.md

```diff
@@ -8,12 +8,122 @@ $ mvn clean install
 
 This will copy the required executables and libraries into \<fido-iot-src\>/demo/owner/.
 
+# Configuring the FIDO IoT Owner Sample
+
+Some required runtime arguments
+
+- `owner_to2_port`
+
+  Owner server port.
+
+  Docker default: 8042
+
+- `owner_database_connection_url`
+
+  JDBC URL for connection to database. Includes the database driver name, port number for database and the location of `.db` file.
+
+  Docker default: jdbc:h2:tcp://localhost:8051/./target/data/ops
+
+- `owner_database_username`
+
+  Owner database username.
+
+  Docker default: sa
+
+- `owner_database_password`
+
+  Owner database password.
+
+  Docker default: <no-password>
+
+- `owner_database_port`
+
+  RV database port number.
+
+  Docker default: 8050
+
+- `epid_online_url`
+
+  EPID Verification Service URL for EPID device signature verification.
+
+  Docker Default: https://verify.epid-sbx.trustedservices.intel.com/
+  Other server options: https://verify.epid.trustedservices.intel.com/ (production EPID verification server), https://localhost:1180 (onprem verification service)
+
+- `catalina_home`
+
+  Tomcat configuration for cataline home.
+
+  Docker default: ./target/tomcat
+
+- `owner_keystore_password`
+
+  Keystore password for owner_keystore.p12 and the internal softHSM's PKCS11 keystore.
+
+  Docker default: OnrKstr1
+
+- `owner_to0_scheduling_enabled`
+
+  Auto completes TO0 for GUIDs with DI complete state.
+
+  Docker default: false
+
+- `owner_to0_scheduling_interval`
+
+  Time interval to check database for GUIDs with pending TO0.
+
+  Docker default: 300s
+
+- `owner_to0_rv_blob`
+
+  Information containing network address of prospective owner. Owner shares this information with RV during TO0. RV, then shares the same during TO1. Device, then uses this information to initiate TO2 protocol.
+
+  Docker default: http://localhost:8042?ipaddress=127.0.0.1
+
+- `owner_api_user'
```

Comment:
Not rendered correctly. Last character is ' instead of `

Reply:
Done